src/views/PreLaunchAirdrop/components/cards/BridgeTravelerCard.tsx

```diff
@@ -35,7 +35,7 @@ function useBridgeTravelerCard(
 
   let cardDescription =
     "Have you bridged before but have yet to use Across? Connect your wallet to check if you're eligible for an airdrop through the Bridge Traveler Program.";
-  if (isConnected) {
+  if (isConnected && isWalletEligible !== undefined) {
     if (isWalletEligible !== false) {
```

Comment:
Maybe we could have just coerced this to a truthy bool like
```suggestion
    if (isWalletEligible) {
```

Reply:
^^ Seems nice to avoid the double check on isWalletEligible, but I'm happy either way.